packages/features/ee/workflows/lib/reminders/providers/twilioProvider.ts

```diff
@@ -6,25 +6,13 @@ import { setTestSMS } from "@calcom/lib/testSMS";
 import prisma from "@calcom/prisma";
 import { SMSLockState } from "@calcom/prisma/enums";
 
-declare global {
```

Comment:
removing the global variable and creating TwilioClient for every function separately. Not sure if that will fix it, but that's the only thing I found that I thought could be the issue